tests/end-to-end/upgrade/pkg/tests/event-bus/event_bus.go

```diff
@@ -98,10 +98,6 @@ func (f *eventBusFlow) createResources() error {
 		f.createSubscription,
 		f.createSubscriber,
 		f.checkSubscriberStatus,
```

Comment:
There is an issue https://github.com/kyma-project/kyma/issues/5056 to add this back again.